piano-hero/core/src/main/java/com/novoda/pianohero/Note.java

```diff
@@ -0,0 +1,39 @@
+package com.novoda.pianohero;
+
+public class Note {
+
+    private static final int LOWEST_MIDI_NOTE = 0;
+    private static final int HIGHEST_MIDI_NOTE = 127;
+
+    private final int midiNoteNumber;
+
+    public Note(int midiNoteNumber) {
```

Comment:
MIDI seems to be a really convenient way of representing the pitch since any of the notes on a piano keyboard can be assigned an integer value.

It doesn't make an assumption as to the size of the piano (or even that you're using a piano) which is cool, and we can even derive the associated pitch in hertz for a given note with the midiNoteNumber! :boom:

![image](https://cloud.githubusercontent.com/assets/2678555/24528418/3971e1b8-159e-11e7-8ce2-1f6e2cbbe27b.png)

https://en.wikipedia.org/wiki/Scientific_pitch_notation#Table_of_note_frequencies